types/chayns/index.d.ts

```diff
@@ -0,0 +1,938 @@
+// Type definitions for chayns 0.14
+// Project: https://gitlab.tobit.com/chayns-api/chayns-typings
```

Comment:
not sure what you mean by "internal". these packages are consumed by everyone, and the urls should be available to all.